src/main/java/com/softserveinc/dokazovi/dto/post/LatestPostDTO.java

```diff
@@ -0,0 +1,19 @@
+package com.softserveinc.dokazovi.dto.post;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.sql.Timestamp;
+
+@Data
+@Builder
+public class LatestPostDTO {
```

Comment:
The class name should follow naming conventions e.g. LatestPostDto

Reply:
@melnikovae87 asked us to call our DTOs

Reply:
Ok, missed that.